README.md

```diff
@@ -119,6 +119,44 @@ interact with the webapp.
 bin/dc-logs -f
 ```
 
+### Database
+
+The database, named `new_to_nl_development` would be accessible on the
+port `5345` in the local host.  By default, the username is `postgres`
+and the password is namesake.  You can explore the database using
+`pssql`:
+
+```bash
+psql -h localhost new_to_nl_development postgres
```

Comment:
This is providing access to the local app db using the postgres shell, right? Cool.

Reply:
Indeed.